### PR TITLE
Integrate Material Design (and state handling for fields) in views: Send, SendInvoke, TestInvoke, Transactions, Balance, Login, Logout

### PR DESCRIPTION
**Pull request by deanpress**

### CODE REVIEW

File: src/app/containers/Send/index.js [hunk 1]
```diff
@@ -42,9 +66,19 @@ export default class Send extends Component {
...
+    if (this.state.assetType === 0) {
+      assetType = 'NEO'
+    } else {
+      assetType = 'GAS'
+    }
+
+    let amounts = {}
+    amounts[assetType] = parseFloat(this.state.amount)
+    console.log('submit')
+    console.log(network.name, this.state.address, account.wif, amounts)
```
Comment (slipo) [inline src/app/containers/Send/index.js]: Might be best to remove these if they were just for personal debugging

File: src/app/components/Login/index.js [hunk 2]
```diff
@@ -60,43 +77,51 @@ export default class Login extends Component {
...
       const myAccount = Neon.create.account(account.wif)
       return (
         <div>
-          <p>Logged In</p>
           <Button ripple raised onClick={this.handleClick}>
             Logout
           </Button>
-          <div>Address: {myAccount.address}</div>
-          <div>Public key encoded: {myAccount.getPublicKey(true)}</div>
+          <div style="margin-top:20px;">
```
Comment (slipo) [inline src/app/components/Login/index.js]: Move inline styles to Login.css ?

**Comment by slipo** [comment]
lgtm and works well. Just a couple small comments.

File: src/app/components/Login/index.js (hunk 2 above)
Comment (deanpress) [reply src/app/components/Login/index.js]: Good find! Updated.

File: src/app/containers/Send/index.js (hunk 1 above)
Comment (deanpress) [reply src/app/containers/Send/index.js]: Good find! Updated.

File: src/app/components/Login/index.js (hunk 2 above)
Comment (deanpress) [reply src/app/components/Login/index.js]: Fixed!

**Comment by slipo** [comment]
lgtm

**Pull request merged**